From 93b6afc62d174759ab0f9ebf5539fed48c9e3973 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Sun, 31 Jul 2011 19:20:02 -0400
Subject: sh: Add export.h to arch/sh specific files as required.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
(cherry picked from commit f7be345515ab6d5c3a0973bb2b32510fcb7c0481)

Conflicts:

	arch/sh/drivers/pci/pci.c
	arch/sh/kernel/cpu/shmobile/cpuidle.c
	arch/sh/kernel/perf_event.c
	arch/sh/kernel/topology.c
	arch/sh/mm/init.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 drivers/sh/intc/core.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/sh/intc/core.c b/drivers/sh/intc/core.c
index f892529..8b7a141 100644
--- a/drivers/sh/intc/core.c
+++ b/drivers/sh/intc/core.c
@@ -30,6 +30,7 @@
 #include <linux/list.h>
 #include <linux/spinlock.h>
 #include <linux/radix-tree.h>
+#include <linux/export.h>
 #include "internals.h"
 
 LIST_HEAD(intc_list);
-- 
1.7.10

